src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -164,6 +164,12 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
+    def listClustersByCreator(creator: WorkbenchEmail): DBIO[Seq[Cluster]] = {
+      clusterQuery.filter { _.creator === creator.value }.result map { recs =>
```

Comment:
If adding this query we should also add an index on the `creator` column -- but as noted below I'm not sure it's needed.